VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/private-consultation-participants.component.ts

```diff
@@ -104,6 +104,18 @@ export class PrivateConsultationParticipantsComponent extends WRParticipantStatu
         return '';
     }
 
+    getParticipantRowClasses(participant: any): string {       
+        if (!this.isJohConsultation()) {
+            return 'participant-row';
+        } 
+        if (this.isJohConsultation() && (!this.isLastJohFromGroupMember(participant))) {            
+            return 'participant-group-row';
```

Comment:
I suspect the isLastJohFromGroupMember stuff can just be acheived with css

Reply:
We need it to determine the participants order on a group.